include/samplog/LogLevel.h

```diff
@@ -14,8 +14,15 @@ enum samplog_LogLevel
 	INFO = 2,
 	WARNING = 4,
 	ERROR = 8,
+	FATAL = 16,
```

Comment:
I feel like a `FATAL` log level should be handled in a special way, something like printing the log line to console or having a `fatal.log` file, just like with the `error.log` and `warning.log` files.
And if a fatal log line comes in, the log core should notice that and shutdown properly, since a fatal error basically means that the application is about to terminate.

I had a `FATAL` log level at some earlier point in development, but decided to remove it, because I couldn't really think of any reason how this is relevant within a SA-MP plugin. Plugins should run stable, and not just terminate because of a known reason or condition.